index.js

```diff
@@ -362,13 +367,80 @@ function filterFormatCodecs(format) {
     && (acodec !== 'none' || (acodec === 'none' && !protocol.includes('https')));
 }
 
-function filterFormatFps(format) {
+function filterVideoFormatFps(format) {
   const { fps, height } = format;
   return ((height >= 1080 && fps <= 30) || height < 1080);
 }
 
+function processAudioFormats(formats) {
+  // Filter out tracks that are not suitable (see comments below)
+  const filteredFormats = formats.filter((format) => filterAudioFormatCodecs(format));
+  // Sort the tracks because .find will return the first match
+  filteredFormats.sort(audioTrackSort)
```

Comment:
What's this sort based on?

Reply:
Based on the audioTrackSort function, which now sorts by
- prefer english tracks over other languages
- prefer https/opus tracks over m3u8 tracks
- prefer higher audio bit rate
- final sort based on format_id so final order is deterministic